zerver/lib/actions.py

```diff
@@ -1049,8 +1056,9 @@ def check_message(sender, client, message_type_name, message_to,
         message.pub_date = timezone.now()
     message.sending_client = client
 
-    if not message.maybe_render_content(realm.domain):
-        raise JsonableError(_("Unable to render message"))
+    # We are going to assume that any message can be rendered, and if it fails,
+    # it will have to fail later.
+    assert message.rendered_content is None
```

Comment:
This commit breaks the error handling for the case that the message isn't renderable, right?  Or am I missing something?